src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -16,11 +17,13 @@ import './gitops.css';
 enum GitProvider {
     GitLab = 'GITLAB',
     Github = 'GITHUB',
+    Azure_devops = 'AZURE_DEVOPS'
 };
 
 const GitHost = {
     GITHUB: "https://github.com/",
-    GITLAB: "https://gitlab.com/"
+    GITLAB: "https://gitlab.com/",
```

Comment:
where is it used?

Reply:
we are using it in case of abstracting out host in  input field
<img width="709" alt="Screenshot 2021-05-20 at 10 26 02 AM" src="https://user-images.githubusercontent.com/61836271/118921560-02c93980-b956-11eb-872c-cd6b1e462f4a.png">